### DISCUSSION TITLE
Report email template customization

### DISCUSSION THREAD

**Discussion by chetank81** [apache/superset#18677] [Q&A / Help]
Hi All,

At the moment so far with my exploration with SuperSet. I did not find any way to customize the email templates. Has someone tried customizing the email templates, any guidance?

--Chetan

**Comment by phifa** [comment]
@chetank81 did you figure this out? I would like to customize the mails too

**Comment by mattitoo** [comment]
I think this is the best place the start: https://github.com/apache/superset/blob/master/superset/reports/notifications/email.py

**Comment by Ayoub-BOUCHACHIA** [comment]
@chetank81 @phifa

Hello, has anyone figured out how to override the email template in Superset for alerting and reporting?
